### PR TITLE
*: refine engineering structure

### PR DESCRIPTION
**Pull request by IANTHEREAL**
* add some READMEs
* refine engineering structure

### CODE REVIEW

**Comment by IANTHEREAL** [comment]
@july2993 @WangXiangUSTC PTAL

**Comment by IANTHEREAL** [comment]
I would adjust `TiDB-Binlog` later

File: tidb_binlog/binlog_driver/example/mysql/mysql.go [hunk 1]
```diff
@@ -22,8 +22,8 @@ import (
 	"github.com/Shopify/sarama"
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/ngaut/log"
-	"github.com/pingcap/tidb-tools/binlog_driver/reader"
-	pb "github.com/pingcap/tidb-tools/binlog_proto/go-binlog"
+	"github.com/pingcap/tidb-tools/tidb_binlog/binlog_driver/reader"
```
Comment (july2993) [inline tidb_binlog/binlog_driver/example/mysql/mysql.go]: s/binlog_driver/driver/ ?  
if it's a sub path of tidb_binlog now

**Review by july2993 — COMMENTED** [review]
rest LGTM

File: tidb_binlog/binlog_driver/example/mysql/mysql.go (hunk 1 above)
Comment (IANTHEREAL) [reply tidb_binlog/binlog_driver/example/mysql/mysql.go]: ok

**Comment by WangXiangUSTC** [comment]
LGTM

**Pull request merged**